lms/djangoapps/teams/views.py

```diff
@@ -165,7 +165,7 @@ def get(self, request, course_id):
             is_user_org_protected = organization_protection_status == OrganizationProtectionStatus.protected
             filter_query['organization_protected'] = is_user_org_protected
 
-        user_teams = CourseTeam.objects.filter(**filter_query)
+        user_teams = CourseTeam.objects.filter(**filter_query).order_by('-last_activity_at', 'team_size')
```

Comment:
Curious, why are we also sorting by team_size? Otherwise looks good.

Reply:
Team size is the tie-breaker that's used in other places in the code... not sure why it was chosen but trying to be consistent. 

Good question, though...